fasm/model.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2017-2020  The SymbiFlow Authors.
+#
+# Use of this source code is governed by a ISC-style
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/ISC
+#
+# SPDX-License-Identifier: ISC
+
+from collections import namedtuple
+import enum
+
+
+# Number format used for a FASM value.
```

Comment:
These comments should be Python docstrings.

How you document enums is a little tricky.
 - https://stackoverflow.com/questions/52062831/how-do-i-properly-document-python-enum-elements
 - https://github.com/microsoft/python-language-server/issues/1725

Reply:
Docstrings for enum don't seem to be standardized yet. I don't think adding a dependency (`aenum`) just to document this enum makes sense.

Reply:
@litghost What do you think about this?

Reply:
I've set the `__doc__` values, which isn't too verbose and works on Python 3.x without additional dependencies.